low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -23,39 +23,41 @@ import sys
 import socket
 import json
 import subprocess
+import tempfile
+import warnings
+warnings.filterwarnings(action='ignore',module='cryptography')
 
-from cortx.utils.conf_store import Conf
-from framework.utils.conf_utils import Utility
-from framework.base.sspl_constants import (PRODUCT_FAMILY,
-                                        RESOURCE_HEALTH_VIEW,
-                                        sspl_config_path,
-                                        GLOBAL_CONFIG_INDEX)
+from framework.utils.conf_utils import (GLOBAL_CONF, Conf,
+    STORAGE_TYPE_KEY, SITE_ID_KEY, RACK_ID_KEY,
+    NODE_ID_KEY, CLUSTER_ID_KEY)
+from framework.base.sspl_constants import PRODUCT_FAMILY, DATA_PATH
 
-SSPL_CONF = "SSPL_CONFIG"
-Conf.load(SSPL_CONF, sspl_config_path)
-global_config_url = Conf.get(SSPL_CONF, "SYSTEM_INFORMATION>global_config_copy_url")
-Conf.load(GLOBAL_CONFIG_INDEX, global_config_url)
-
-machine_id = Utility.get_machine_id()
-
-# Get cluster_id and node_key_id from global config
-cluster_id = Conf.get(GLOBAL_CONFIG_INDEX, "server_node>%s>cluster_id" % machine_id)
-node_key_id = Conf.get(GLOBAL_CONFIG_INDEX, "server_node>%s>name" % machine_id)
-
-onGoingRequestPath = "/tmp/sspl-msb-req.txt"
+onGoingRequestPath = "f{DATA_PATH}sspl-msb-req.txt"
+sspl_state_file = f"/var/{PRODUCT_FAMILY}/sspl/data/state.txt"
 
 class ManifestSupportBundle(object):
 
+    STORAGE_TYPE_RBOD = 'rbod'
+
     def __init__(self):
 
         self.localTempPath = "/tmp/support_bundle/"
-        self.encl_manifest = "/tmp/encl_manifest.json"
-        self.node_manifest = "/tmp/node_manifest.json"
-        self.manifest_summary = "/tmp/manifest_summary.json"
+        self.encl_manifest = f"{DATA_PATH}encl_manifest.json"
```

Comment:
Add files in a specific folder(create if needed) for manifest, health, as applicable.

Reply:
Done